packages/headless/src/features/commerce/facets/facet-set/facet-set-slice.test.ts

```diff
@@ -67,10 +67,7 @@ import {
   updateDateFacetValues,
 } from '../date-facet/date-facet-actions.js';
 import {getFacetIdWithCommerceFieldSuggestionNamespace} from '../facet-search-set/commerce-facet-search-actions.js';
-import {
```

Comment:
My understanding is that we don't believe it really makes sense to support excluding a store from location selection. @pvanool @BenKandelaars feel free to expand on this! 😇

Reply:
Upon further thinking, I believe I understand the reason.

Let's say an org has three locations, A, B, and C. Each product can be available in one or more of those locations.

A user could legimately want to see only products that are available in location A or B (the locations that are closest to him). However, there's no reason for them to exclude products that are available in location C, because these products might also be available in locations A or B.

Reply:
Yes, this exactly. We can't see a use case for it at the moment. And also since it's not a "regular" facet we decided not to explore it for the initial delivery.